modules/cart/src/Tests/AddToCartFormTest.php

```diff
@@ -288,10 +271,32 @@ public function testExposedLineItemFields() {
     // Check if the quantity was increased for the existing line item.
     $this->cart = Order::load($this->cart->id());
     $line_items = $this->cart->getLineItems();
-    /** @var \Drupal\commerce_order\Entity\LineItemInterface $line_item */
-    $line_item = $line_items[0];
-    $this->assertEqual($line_item->getTitle(), $this->variation->getLineItemTitle());
-    $this->assertTrue(($line_item->getQuantity() == 3), t('The product @product has been added to cart.', ['@product' => $line_item->getTitle()]));
+
+    $this->assertLineItemInOrder($this->variation, $line_items[0], 3);
+  }
+
+  /**
+   * Creates an attribute field and set of attribute values.
+   *
+   * @param \Drupal\commerce_product\Entity\ProductVariationTypeInterface $variation_type
+   *   The variation type.
+   * @param string $name
+   *   The attribute field name.
+   * @param array $options
+   *   Associative array of key name values. [red => Red].
+   *
+   * @return \Drupal\Core\Entity\EntityInterface[]
+   *   Array of attribute entities.
+   */
+  protected function createAttributeSet(ProductVariationTypeInterface $variation_type, $name, array $options) {
```

Comment:
Should this and some of its helpers be traits for other product tests, @bojanz? Could put into its own PR